css/fluo.scss

```diff
@@ -87,6 +87,13 @@ tr:nth-child(odd) {background: #FFF}
   margin-top: 20px;
 }
 
+/* Turn off sidebar affix for small screens */
+@media (max-width: 975px) {
```

Comment:
Will this work for small screens that are 4K?  Like a 4K phone?

Reply:
I am not sure.  It needs be tested.